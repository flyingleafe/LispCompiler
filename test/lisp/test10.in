12345
-11223344