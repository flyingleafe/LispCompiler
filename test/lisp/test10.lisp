;; Calling outer functions
(printInt 5)
