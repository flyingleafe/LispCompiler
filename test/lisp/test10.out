5
12345
-10
-11223344
